charts/backstage/values.yaml

```diff
@@ -0,0 +1,141 @@
+GreplicaCount: 1
+
+image:
+  #repository: ghcr.io/raffaelespazzoli/backstage-autobuild
+  pullPolicy: Always
+  #tag: "main"
+  repository: quay.io/assemble/backstage
+  tag: "55a870a1b99a04f7b22339804a5a2b7ed1c4ff58"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: ClusterIP
+  port: 8080
+  targetPort: 7007
+
+## For custom route provide value
+route:
+  enabled: true
+
+ingress:
+  enabled: false
+
+  ## @param ingress.className Name of the IngressClass cluster resource which defines which controller will implement the resource (e.g nginx)
+  className: ""
+
+  ## @param ingress.annotations Additional annotations for the Ingress resource
+  annotations: 
+    nginx.ingress.kubernetes.io/rewrite-target: "/"
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+    route.openshift.io/termination: "edge"
+
+  ## @param host Hostname to be used to expose the route to access the backstage application (e.g: backstage.IP.nip.io)
+  host: ""
```

Comment:
I think `host` value should be the same/match `.backstage.baseUrl` shouldn't it? If they differ backstage won't be able to perform auth properly/cors properly.

https://backstage.io/docs/auth/troubleshooting#auth-fails-with-login-failed-caused-by-notallowederror-origin--is-not-allowed

Reply:
Modified to only use the baseUrl for all host vars